app/services/settings_service/repository_base.rb

```diff
@@ -30,7 +30,7 @@ def create_table(name:)
     end
 
     def dynamodb
-      @dynamodb ||= Rails.env.development? ? Aws::DynamoDB::Client.new(:endpoint => "http://dynamodb:8000/") : Aws::DynamoDB::Client.new
+      @dynamodb ||= Rails.env.development? ? Aws::DynamoDB::Client.new(:endpoint => "http://localhost:8000/") : Aws::DynamoDB::Client.new
```

Comment:
we should extract this into an env variable.

```suggestion
      @dynamodb ||= Rails.env.development? ? Aws::DynamoDB::Client.new(:endpoint => "http://#{ENV['DYNAMO_DB_SERVER']}:8000/") : Aws::DynamoDB::Client.new
```

Reply:
@adiaz04 - would that be added to the `canvas.env` file that doesn't get checked into source control or somewhere else?

Reply:
@CJAdeszko I thought we had done this already but I guess it didn't get checked in. DYNAMO_DB_SERVER is in my `canvas.env` - the one that doesn't get checked in. So I'm guessing yes. But I had the full URL in mine: `DYNAMO_DB_SERVER=http://localhost:8000/`

Reply:
@adiaz04 any reason we don't want the full URL in there?

Reply:
Full URL is also fine.

Reply:
We're going to make this change with QTY-1599 so we don't have to rebase this again